DigitalOffice/Services/UserService/Startup.cs

```diff
@@ -46,16 +46,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         private static void UpdateDatabase(IApplicationBuilder app)
         {
```

Comment:
Remove "static" modifier as we discussed it on the call. Other that that all's OK